packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -446,15 +446,22 @@ contract Deploy is Deployer {
     /// @notice Deploys an ERC1967Proxy contract with the ProxyAdmin as the owner.
     /// @param _name The name of the proxy contract to be deployed.
     /// @return addr_ The address of the deployed proxy contract.
-    function deployERC1967Proxy(string memory _name) public broadcast returns (address addr_) {
+    function deployERC1967Proxy(string memory _name) public returns (address addr_) {
         addr_ = deployERC1967ProxyWithOwner(_name, mustGetAddress("ProxyAdmin"));
     }
 
     /// @notice Deploys an ERC1967Proxy contract with a specified owner.
     /// @param _name The name of the proxy contract to be deployed.
     /// @param _proxyOwner The address of the owner of the proxy contract.
     /// @return addr_ The address of the deployed proxy contract.
-    function deployERC1967ProxyWithOwner(string memory _name, address _proxyOwner) public returns (address addr_) {
+    function deployERC1967ProxyWithOwner(
+        string memory _name,
+        address _proxyOwner
+    )
+        public
+        broadcast
```

Comment:
Moving `broadcast` to here from the above location ensures that the on-chain action is always wrapped in `vm.broadcast()`.